integration-tests/utils/integration-utils.js

```diff
@@ -169,10 +169,15 @@ const dismissNotifications = async page => {
   return !!notificationCloseButtons.length && delay(1000) // delayed for alerts to animate off
 }
 
-const signIntoTerra = async (page, token) => {
-  await page.waitForXPath('//*[contains(normalize-space(.),"Loading Terra")]', { hidden: true })
+const signIntoTerra = async (page, { token, testUrl }) => {
+  if (testUrl) {
+    await page.goto(testUrl, { waitUntil: ['load', 'domcontentloaded', 'networkidle0'], timeout: 60 * 1000 })
+  }
```

Comment:
What's the semantics of the predicate (`if (testUrl)`) here? When is a caller of `signIntoTerra` supposed to pass in a `testUrl` and when not?